runtime/centrifuge/src/migrations.rs

```diff
@@ -181,57 +193,413 @@ mod asset_registry {
+				existential_deposit: 10_000u128.into(),
+				location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+					1,
+					Junctions::X3(Parachain(1000), PalletInstance(50), GeneralIndex(1984)),
+				))),
+				additional: CustomMetadata {
+					mintable: false,
+					permissioned: false,
+					pool_currency: true,
+					transferability: CrossChainTransferability::Xcm(XcmMetadata {
+						fee_per_second: None,
+					}),
+				}
+				.into(),
+			},
+		),
+		(
+			CurrencyId::ForeignAsset(2),
+			orml_asset_registry::AssetMetadata {
+				decimals: 6,
+				name: b"Axelar USDC".to_vec(),
+				symbol: b"xcUSDC".to_vec(),
+				existential_deposit: 10_000u128.into(),
+				location: None,
+				additional: CustomMetadata {
+					mintable: false,
+					permissioned: false,
+					pool_currency: true,
+					transferability: CrossChainTransferability::Xcm(XcmMetadata {
+						fee_per_second: None,
+					}),
+				}
+				.into(),
+			},
+		),
+		(
+			CurrencyId::ForeignAsset(3),
+			orml_asset_registry::AssetMetadata {
+				decimals: 12,
+				name: b"Acala Dollar".to_vec(),
+				symbol: b"aUSD".to_vec(),
+				existential_deposit: 10_000_000_000u128.into(),
+				location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+					1,
+					Junctions::X2(
+						Parachain(2000),
+						GeneralKey {
+							length: 2,
+							data: gk,
+						},
+					),
+				))),
+				additional: CustomMetadata {
+					mintable: false,
+					permissioned: false,
+					pool_currency: true,
+					transferability: CrossChainTransferability::Xcm(XcmMetadata {
+						fee_per_second: None,
+					}),
+				}
+				.into(),
+			},
+		),
+		(
+			CurrencyId::ForeignAsset(4),
+			orml_asset_registry::AssetMetadata {
+				decimals: 18,
+				name: b"Glimmer".to_vec(),
+				symbol: b"GLMR".to_vec(),
+				existential_deposit: 1_000_000_000_000_000u128.into(),
+				location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+					1,
+					Junctions::X2(Parachain(2004), PalletInstance(10)),
+				))),
+				additional: CustomMetadata {
+					mintable: false,
+					permissioned: false,
+					pool_currency: false,
+					transferability: CrossChainTransferability::Xcm(XcmMetadata {
+						fee_per_second: None,
+					}),
+				}
+				.into(),
+			},
+		),
+		(
+			CurrencyId::ForeignAsset(5),
+			orml_asset_registry::AssetMetadata {
+				decimals: 10,
+				name: b"DOT".to_vec(),
+				symbol: b"DOT".to_vec(),
+				existential_deposit: 100_000u128.into(),
+				location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+					1,
+					Junctions::Here,
+				))),
+				additional: CustomMetadata {
+					mintable: false,
+					permissioned: false,
+					pool_currency: false,
+					transferability: CrossChainTransferability::Xcm(XcmMetadata {
+						fee_per_second: None,
+					}),
+				}
+				.into(),
+			},
+		),
+		// Adding LP USDC here
+		(
+			LP_ETH_USDC_CURRENCY_ID,
+			lp_eth_usdc_metadata(
+				LiquidityPoolsPalletIndex::get(),
+				ETHEREUM_MAINNET_CHAIN_ID,
+				ETHEREUM_USDC,
+			),
+		),
+	]
```

Comment:
Add here any current registered asset that is added between now and the upgrade.